<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReliabilityAnalysisForm</class>
 <widget class="QWidget" name="ReliabilityAnalysisForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <widget class="QWidget" name="topWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="2" rowspan="3">
       <widget class="BoundListView" name="variables">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="AssignButton" name="assignButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>81</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>81</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" rowspan="3">
       <widget class="AvailableFieldsListView" name="listAvailableVariables">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="gridLayout_7" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="BoundGroupBox" name="totalScaleStats">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Scale Statistics</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="0">
          <widget class="BoundCheckBox" name="averageInterItemCor">
           <property name="text">
            <string>Average interitem correlation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="BoundCheckBox" name="alphaScale">
           <property name="text">
            <string>Cronbach's α</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="BoundCheckBox" name="sdScale">
           <property name="text">
            <string>Standard deviation</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="BoundCheckBox" name="meanScale">
           <property name="text">
            <string>Mean</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="BoundCheckBox" name="gutmannScale">
           <property name="text">
            <string>Gutmann's λ6</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="BoundCheckBox" name="mcDonaldScale">
           <property name="text">
            <string>McDonalds' ω</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="BoundGroupBox" name="itemStats">
        <property name="title">
         <string>Individual Item Statistics</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="5" column="0">
          <widget class="BoundCheckBox" name="itemRestCor">
           <property name="text">
            <string>Item-rest correlation</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="BoundCheckBox" name="gutmannItem">
           <property name="text">
            <string>Gutmann's λ6 (if item dropped)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="BoundCheckBox" name="sdItem">
           <property name="text">
            <string>Standard deviation</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="BoundCheckBox" name="alphaItem">
           <property name="text">
            <string>Cronbach's α (if item dropped)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="BoundCheckBox" name="meanItem">
           <property name="text">
            <string>Mean</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="BoundCheckBox" name="mcDonaldItem">
           <property name="text">
            <string>McDonalds' ω (if item dropped)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="revScaledItemsBox" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item row="1" column="0">
       <widget class="QWidget" name="containerRevScaledItems" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="BoundAssignWidget" name="reverseScaledItems" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>160</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="ExpanderButton" name="expanderPosthoc">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Reverse-Scaled Items</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AssignButton</class>
   <extends>QPushButton</extends>
   <header>widgets/assignbutton.h</header>
  </customwidget>
  <customwidget>
   <class>AvailableFieldsListView</class>
   <extends>QListView</extends>
   <header>widgets/availablefieldslistview.h</header>
  </customwidget>
  <customwidget>
   <class>BoundCheckBox</class>
   <extends>QCheckBox</extends>
   <header>widgets/boundcheckbox.h</header>
  </customwidget>
  <customwidget>
   <class>BoundListView</class>
   <extends>QListView</extends>
   <header>widgets/boundlistview.h</header>
  </customwidget>
  <customwidget>
   <class>BoundGroupBox</class>
   <extends>QGroupBox</extends>
   <header>widgets/boundgroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExpanderButton</class>
   <extends>QPushButton</extends>
   <header>widgets/expanderbutton.h</header>
  </customwidget>
  <customwidget>
   <class>BoundAssignWidget</class>
   <extends>QWidget</extends>
   <header>widgets/boundassignwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
